### DISCUSSION TITLE
Adding custom queries

### DISCUSSION THREAD

**Discussion by ranjithshegde** [nvim-orgmode/orgmode#233] [Q&A]
Since setting [conceal](https://github.com/neovim/neovim/pull/17776) from treesitter has been merged in core, I have been trying to set my own queries for org bullets like this
```query
(
((stars) @stars (#eq? @stars "*")) @_conceal
(#set! conceal "◉")
)
(
((stars) @stars (#eq? @stars "**")) @_conceal
(#set! conceal "○")
)
(
((stars) @stars (#eq? @stars "***")) @_conceal
(#set! conceal "✸")
)
(
((stars) @stars (#eq? @stars "****")) @_conceal
(#set! conceal "✿")
)
```
When I append the above snippet to `packer/*/orgmode/queries/org/highlights.scm` it works perfectly. However when I do it in my `~/.config/nvim/after/queries/org/highlights.scm` it does not apply. 
I have tested similar conceal in config path for markdown and it works. 
Does orgmode override the queries?

**Comment by kristijanhusak** [comment]
Oh nice, i wasn't even aware of it. There are only some additional queries loaded for todo keywords, but stars shouldn't be overwritten. Does it happen if you try to override some "Normal" highlight?

**Comment by ranjithshegde** [comment]
I tried setting conceal via after/syntax/ directory like this
```vim
syntax match OrgHeadlineStar1 /^\*\ze\s/me=e-1 conceal cchar=◉ containedin=OrgHeadlineLevel1 contained
```
Same. It does not work

**Reply by kristijanhusak** [reply]
Do you have `additional_vim_regex_highlighting = {'org'}` in your treesitter config ?

**Reply by ranjithshegde** [reply]
Yes i do. is that an issue? Just tried with it off and on. Same!

EDIT: If you are on the latest nightly, would you mind trying to reproduce? using the highlights queries snippet from the first post?

**Reply by kristijanhusak** [reply]
No, you need to have that in order to get old Vim syntax working.
I'm assuming [this](https://github.com/nvim-orgmode/orgmode/blob/master/lua/orgmode/init.lua#L27) is causing the issue. Could you try comment that out and see if it works then? I'm still not updated to the latest Neovim.

**Reply by ranjithshegde** [reply]
Indeed that works!  I do lose todo highlights (of course!) but this works for now!

**Reply by kristijanhusak** [reply]
Cool. Open up an issue for this, and I'll see if there is a way to address it somehow. If not, I can add a hook function which can be triggered after todo highlights are set
